From cfe465d54520cd2fa9d7b8d69cf5c597ebbb9f71 Mon Sep 17 00:00:00 2001
From: Raphael Mounier <mounierr07@gmail.com>
Date: Mon, 31 Mar 2025 13:14:04 +0200
Subject: [PATCH] Revert "SystemUI: Don't shift KeyguardSecurityContainer  when
 FOD isn't available"

[PATCH 09/14] Revert "SystemUI: Don't shift KeyguardSecurityContainer
 when FOD isn't available"

This reverts commit d73dedec31d9b1234beeb146e799691651d08c75.

Change-Id: I175398441c5818e63ba188a28dab832e5ceb9912
---
 .../android/keyguard/KeyguardSecurityContainer.java   | 11 ++---------
 1 file changed, 2 insertions(+), 9 deletions(-)

diff --git a/packages/SystemUI/src/com/android/keyguard/KeyguardSecurityContainer.java b/packages/SystemUI/src/com/android/keyguard/KeyguardSecurityContainer.java
index 27aef8089a8..4cf8c830a34 100644
--- a/packages/SystemUI/src/com/android/keyguard/KeyguardSecurityContainer.java
+++ b/packages/SystemUI/src/com/android/keyguard/KeyguardSecurityContainer.java
@@ -81,8 +81,6 @@ import com.android.systemui.statusbar.policy.DeviceProvisionedController;
 import com.android.systemui.statusbar.policy.KeyguardStateController;
 import com.android.systemui.util.InjectionInflationController;
 
-import lineageos.app.LineageContextConstants;
-
 import java.util.List;
 
 public class KeyguardSecurityContainer extends FrameLayout implements KeyguardSecurityView {
@@ -128,7 +126,6 @@ public class KeyguardSecurityContainer extends FrameLayout implements KeyguardSe
     private InjectionInflationController mInjectionInflationController;
     private boolean mSwipeUpToRetry;
     private AdminSecondaryLockScreenController mSecondaryLockScreenController;
-    private boolean mHasFod;
 
     private final ViewConfiguration mViewConfiguration;
     private final SpringAnimation mSpringAnimation;
@@ -269,9 +266,6 @@ public class KeyguardSecurityContainer extends FrameLayout implements KeyguardSe
                 mUpdateMonitor, mCallback, new Handler(Looper.myLooper()));
         mDeviceProvisionedController = Dependency.get(DeviceProvisionedController.class);
 
-        PackageManager packageManager = mContext.getPackageManager();
-        mHasFod = packageManager.hasSystemFeature(PackageManager.FEATURE_FINGERPRINT) &&
-                packageManager.hasSystemFeature(LineageContextConstants.Features.FOD);
     }
 
     public void setSecurityCallback(SecurityCallback callback) {
@@ -528,9 +522,8 @@ public class KeyguardSecurityContainer extends FrameLayout implements KeyguardSe
 
         // Consume bottom insets because we're setting the padding locally (for IME and navbar.)
         int inset;
-        int minBottomMargin = mHasFod && mUpdateMonitor.isFingerprintDetectionRunning() ?
-                getResources().getDimensionPixelSize(
-                        R.dimen.kg_security_container_min_bottom_margin) : 0;
+        int minBottomMargin = getResources().getDimensionPixelSize(
+            R.dimen.kg_security_container_min_bottom_margin);
 
         if (sNewInsetsMode == NEW_INSETS_MODE_FULL) {
             int bottomInset = insets.getInsetsIgnoringVisibility(systemBars()).bottom;
-- 
2.25.1

